game/assets/jak1/text/game_text_en.gs

```diff
@@ -8,12 +8,16 @@
         "CAMERA OPTIONS")
 (#x1001 "NORMAL"
         "NORMAL")
-(#x1002 "INVERTED"
-        "INVERTED")
-(#x1003 "HORIZONTAL CAMERA CONTROL"
-        "HORIZONTAL CAMERA CONTROL")
-(#x1004 "VERTICAL CAMERA CONTROL"
-        "VERTICAL CAMERA CONTROL")
+(#x1002 "FLIPPED"
+        "FLIPPED")
+(#x1003 "1ST-PERSON HORIZONTAL CAM"
+        "1ST-PERSON HORIZONTAL CAM")
+(#x1004 "1ST-PERSON VERTICAL CAM"
+        "1ST-PERSON VERTICAL CAM")
+(#x1005 "3RD-PERSON HORIZONTAL CAM"
+        "3RD-PERSON HORIZONTAL CAM")
+(#x1006 "3RD-PERSON VERTICAL CAM"
+        "3RD-PERSON VERTICAL CAM")
```

Comment:
now see if you can have "CAMERA" instead of "CAM"! If not just reduce the scale a tiny bit?